### PR TITLE
CHAIN-507: back-to-back simplification and UI changes

### PR DESCRIPTION
**Pull request by bholzman**

### CODE REVIEW

**Review by bflood366 — APPROVED** [review]
looks great - so much cleaner not doing all the dry run estimation stuff.

File: backend/src/main/kotlin/xyz/funkybit/apps/ring/BlockProcessor.kt [hunk 1]
```diff
@@ -69,7 +70,7 @@ class BlockProcessor(
                     }
 
                     for (blockNumber in blocksToProcess) {
-                        logger.info { "Processing block $blockNumber, chainId=$chainId" }
+                        if (!quietMode) logger.info { "Processing block $blockNumber, chainId=$chainId" }
```
[the remark this answers is not in the document]
Comment (bholzman) [reply backend/src/main/kotlin/xyz/funkybit/apps/ring/BlockProcessor.kt:73]: created a `infoVerbose` extension method

File: backend/src/main/kotlin/xyz/funkybit/core/repeater/tasks/RepeaterBaseTask.kt [hunk 2]
```diff
@@ -29,7 +30,7 @@ abstract class RepeaterBaseTask(
         if (isLockAcquired) {
             val startTime = Clock.System.now()
             try {
-                logger.debug { "Acquired lock for ${this.javaClass.simpleName}" }
+                if (!quietMode) logger.debug { "Acquired lock for ${this.javaClass.simpleName}" }
```
[the remark this answers is not in the document]
Comment (bholzman) [reply backend/src/main/kotlin/xyz/funkybit/core/repeater/tasks/RepeaterBaseTask.kt:33]: removed these debugs

File: integrationtests/src/main/kotlin/xyz/funkybit/integrationtests/utils/Wallet.kt [hunk 3]
```diff
@@ -262,16 +262,29 @@ class Wallet(
     }.id
 
     override fun signOrder(request: CreateOrderApiRequest.BackToBackMarket): CreateOrderApiRequest.BackToBackMarket {
-        val (baseSymbol, _) = marketSymbols(request.marketId)
-        val (_, quoteSymbol) = marketSymbols(request.secondMarketId)
+        val market1Symbols = marketSymbols(request.marketId)
```
[the remark this answers is not in the document]
Comment (bholzman) [reply integrationtests/src/main/kotlin/xyz/funkybit/integrationtests/utils/Wallet.kt:265]: yeah, seems fairly orthogonal, going to leave it as is, we can always implement it later if we need it

**Pull request merged**